src/modules/interest-rates/interest-rates.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+
+import { InterestRatesEntity } from './entities/interest-rate.entity';
+
+@Injectable()
+export class InterestRatesService {
+  constructor(
+    @InjectRepository(InterestRatesEntity, 'mssql-cis')
+    private readonly interestRates: Repository<InterestRatesEntity>,
+  ) {}
+
+  async findAll() {
+    try {
+      const response = await this.interestRates.find();
```

Comment:
DWR_INT_RATE table has history or all rates. In DEV environment we have 56462 records, not sure how this be handled here.

To return just active rates we need to filter with column EFFECTIVE_TO_DATETIME = '99991231' , not sure yet how to add it.

Reply:
good shout - I've added the filter now